django/template/engine.py

```diff
@@ -90,6 +104,15 @@ def template_context_processors(self):
         context_processors += tuple(self.context_processors)
         return tuple(import_string(path) for path in context_processors)
 
+    def get_template_builtins(self, builtins):
+        return [import_library(x) for x in builtins]
+
+    def get_template_libraries(self, libraries):
+        loaded = {}
+        for name, path in libraries.items():
+            loaded[name] = import_library(path)
+        return loaded
+
     @cached_property
```

Comment:
We have a small inconsistency here.

Loaders are imported and initialized when the first template is loaded (through a cached property).

Builtins and libraries are imported and initialized when the engine is initialized (and assigned to an attribute).

You pointed out this difference in your description of the PR but you didn't explain the reasoning — or I missed it.

If there's a good reason, perhaps it applies to templates loaders too, and the `template_loaders` property should be refactored into a regular attribute in a follow-up commit?